src/example/java/example/CsvReaderExample.java

```diff
@@ -77,6 +77,7 @@ private static void advancedConfiguration() {
             .commentCharacter('#')
             .skipEmptyRows(true)
             .errorOnDifferentFieldCount(false)
+            .ignoreInvalidQuoteChars(true) // no invalid quote chars in test data, just to see performance difference
```

Comment:
Wondering if this could be a configurable quote escape character instead, if it is not an auto-detect feature.